webui/react/src/pages/WebhookList.tsx

```diff
@@ -83,6 +103,14 @@ const WebhooksView: React.FC = () => {
     fetchWebhooks();
   }, [fetchWebhooks]);
 
+  useEffect(() => {
+    if (settings.workspace?.length) {
```

Comment:
I'm not sure about this, because we use an empty array as default value, if we choose to use null or undefined as default value, we'd have to reset when the array becomes empty.

Reply:
Sure, but this feels a bit unsafe for me, in case the `settings.workspace` is set to 0 accidentally..

Reply:
Up to you